Change archived asciinema example

ericmjalbert opened the pull request:
This fixes #161

pjbull commented:
Thanks @ericmjalbert! 

I see missing characters in the screenshot (probably from powerline?).

<img width="1373" alt="image" src="https://user-images.githubusercontent.com/1799186/56436267-17770700-6290-11e9-981f-af1bd89e8811.png">

Can you also change the asciinema reference in the docs too?

ericmjalbert commented:
@pjbull I updated the asciinema example. I'm not sure why powerline was giving weird characters, but there were also problems with the commands getting repeated twice (`lstotal 0` and `cookiecutterYou've downloaded...` from your screenshot). I think its cause I used `tmux`? Either way this one is simpler and not buggy. I also updated the example in `docs/docs/index.md`. 

I'd be glad to apply any other recommendations!

pjbull commented:
@ericmjalbert Thanks!
